pkgs/by-name/ob/objfw/package.nix

```diff
@@ -30,6 +32,47 @@ clangStdenv.mkDerivation (finalAttrs: {
 
   doCheck = true;
 
+  passthru.tests = {
+    build-hello-world = clangStdenv.mkDerivation {
+      name = "ObjFW test";
+      buildInputs = [ objfw ];
+
+      src = writeTextDir "helloworld.m" ''
+        #import <ObjFW/ObjFW.h>
+        int main() {
+          OFLog(@"Hello world from objc");
+          return 0;
+        }
+      '';
+
+      buildPhase = ''
+        runHook preBuild
+        clang -o testbinary \
+              -x objective-c -Xclang \
```

Comment:
You should get these from `objfw-config`

Reply:
Yes, there is a FHS patch needed there to work with nix, which is why I'm not using it rn. I'll do that patch in a future update